### PR TITLE
Specify driver name in device constructors

### LINKED ISSUE #215: driver_name is missing from device constructors
4a873844cdeff38ebc5d870f8c8bdfedefa2e8e3 removed filtering by `driver_name` attribute from device constructors. Example:

https://github.com/rhempel/ev3dev-lang-python/commit/4a873844cdeff38ebc5d870f8c8bdfedefa2e8e3?diff=split#diff-52ffd7ac54a3727e62910423b8b5a485L800

As a result, when there is single large motor and single medium motor connected to the system, the following script, without explicitly specifying output ports, is not guaranteed to pick correct devices:
```py
lm = LargeMotor()
mm = MediumMotor()
```

### PR DESCRIPTION
**Pull request by ddemidov**
Fixes #215

### CODE REVIEW

**Comment by ddemidov** [comment]
Just checked, this resolves #215 for me. I guess @rhempel omitted driver names by mistake? Will wait a couple of days before merging in case anybody objects.

File: ev3dev/core.py [hunk 1]
```diff
@@ -893,7 +893,7 @@ class FirgelliL1250Motor(Motor):
 
     def __init__(self, address=None, name_pattern=SYSTEM_DEVICE_NAME_CONVENTION, name_exact=False, **kwargs):
 
-        super(FirgelliL1250Motor, self).__init__(address, name_pattern, name_exact, **kwargs)
+        super(FirgelliL1250Motor, self).__init__(address, name_pattern, name_exact, driver_name=['fi-l12-ev3-50'], **kwargs)
```
Comment (WasabiFan) [inline ev3dev/core.py]: This should be the new driver name (the company is now called Actuonix, or something like that); we might need to update the spec for that. The class name needs to be renamed as well.

Comment (ddemidov) [inline ev3dev/core.py]: ev3dev/ev3dev-lang#168

Comment (ddemidov) [inline ev3dev/core.py]: And c5faf77

**Comment by WasabiFan** [comment]
When you merge, please choose the "squash" option and add a note to the message about renaming Firgelli.

**Pull request merged**